controller/src/main/java/com/softserve/borda/controllers/UserController.java

```diff
@@ -42,24 +42,11 @@
         }
     }
 
-    @GetMapping("getById/{id}")
-    public ResponseEntity<UserSimpleDTO> getUserById(@PathVariable Long id) {
-        try {
-            return new ResponseEntity<>(modelMapper.map(
-                    userService.getUserById(id)
-                    ,
-                    UserSimpleDTO.class), HttpStatus.OK);
-        } catch (CustomEntityNotFoundException e) {
-            log.severe(e.getMessage());
-            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
-        }
-    }
-
-    @GetMapping("{id}")
-    public ResponseEntity<UserSimpleDTO> getUserById(@PathVariable String id) {
+    @GetMapping
+    public ResponseEntity<UserSimpleDTO> getUserByToken(@RequestHeader String authorization) {
```

Comment:
Can you explain how it works and for what does it serve?

Reply:
The idea is to get user by authorization token, rather than by id. So it would not be necessary to pass user id, but rather use JWT token already being passed. This way, we can be sure that the one who is requesting user info is the user himself.
Every request sent to our server (except authorization and registration ones) requires JWT token to be passed in a header in the following format:
"Authorization": "Bearer {token}". Then we pass the string "Bearer {token}" into JWTConverter, where token gets extracted and used to receive corresponding user from our database.